packages/react-dom/src/__tests__/ReactDOMFloat-test.js

```diff
@@ -4398,7 +4398,7 @@ background-color: green;
           </body>
         </html>,
       );
-      expect(Scheduler).toFlushWithoutYielding();
+      await waitForAll([]);
```

Comment:
whoops must have snuck one back in :)

Reply:
I think its just because I forgot to include `toFlushWithoutYielding` in the original codemod. I had also originally written it to check for ExpressionStatements instead of CallExpressions so I missed some of those, too.